File: app/src/main/java/com/semisonfire/cloudgallery/data/remote/response/DiskResponse.java
package com.semisonfire.cloudgallery.data.remote.response;

import com.google.gson.annotations.SerializedName;
import com.semisonfire.cloudgallery.data.model.Photo;

import java.util.List;

public class DiskResponse {

    @SerializedName("items")
    private List<Photo> photos = null;

    public List<Photo> getPhotos() {
        return photos;
    }

    public void setPhotos(List<Photo> photos) {
        this.photos = photos;
    }

}

File: app/src/main/java/com/semisonfire/cloudgallery/data/remote/response/DiskResponse.kt
package com.semisonfire.cloudgallery.data.remote.response

import com.google.gson.annotations.SerializedName
import com.semisonfire.cloudgallery.data.model.Photo

class DiskResponse {
  @SerializedName("items")
  var photos: List<Photo>? = null
}